src/pages/workspaces/List.js

```diff
@@ -343,7 +336,7 @@ export const WorkspaceList = () => {
             placeholder: 'Access levels',
             'aria-label': 'Filter by access levels',
             value: accessLevelsFilter,
-            onChange: data => setAccessLevelsFilter(_.map('value', data)),
+            onChange: data => Nav.updateSearch({ ...query, accessLevelsFilter: _.map('value', data) }),
```

Comment:
I don't love how these array values (this an access level) end up serializing:
![Screen Shot 2022-06-15 at 2 48 51 PM](https://user-images.githubusercontent.com/19228696/173902774-326bb680-e692-407f-94a2-36458c782dd8.png)
Is there a way we can do away with the array indexes and instead use multi-valued attributes, inferring ordering by order in the query string? Do you think it's not worth the effort to try?

Also, fresh off the security training, this is one of those serialization implementation details that it mentions watching out for (though I don't think there's really a problem here).

Reply:
> Is there a way we can do away with the array indexes and instead use multi-valued attributes, inferring ordering by order in the query string? Do you think it's not worth the effort to try?

So instead of `list[0]=foo&list[1]=bar`, `list[]=foo&list[]=bar`? Or `list=foo&list=bar`? The library we use for handling query strings does allow controlling how arrays are serialized.
https://www.npmjs.com/package/qs#user-content-stringifying

Reply:
I like `list=foo&list=bar` and I think that's the most natural representation of the data in a query string.

Reply:
After trying `list=foo&list=bar a`, it complicates the parsing logic because there's nothing hinting that `list=foo` should be parsed into an array with one item vs a string.

Reply:
Ah, interesting point. Oh well.

Reply:
Changed from `list[0]=foo&list[1]=bar` to `list[]=foo&list[]bar` in [3ad7337](https://github.com/DataBiosphere/terra-ui/pull/3137/commits/3ad7337a0368817c0b2545550642f8deddb2e5fa).